test/fixtures/deploy/assets-yui.json

```diff
@@ -1 +1 @@
4d3a7c48867c75e2485f.png", "sri":"sha256-T9PQekCQeDFHwAxdZMQs/QzAqaqtimo0uLCotJ0xSqg="}, "/assets/admin/ui-bg_flat_10_000000_40x100.png":{"target":"/assets/admin/ui-bg_flat_10_000000_40x100-c31d5fc3eb7d82c628a82e3b87024cd1.png", "sri":"sha256-r1CSqnKXLAw+qLRMm5LbIAnqQWB3OLzAqdpZesaoc+o="}, "/assets/admin/ui-bg_glass_100_f6f6f6_1x400.png":{"target":"/assets/admin/ui-bg_glass_100_f6f6f6_1x400-c17f552e8f4697d7608c57653af36df0.png", "sri":"sha256-/ws/N6fYaeQspm2VHS5Wm+QV4CXhlUoSqGQ+lmW3xm0="}, "/assets/admin/ui-bg_glass_100_fdf5ce_1x400.png":{"target":"/assets/admin/ui-bg_glass_100_fdf5ce_1x400-fe58c3539111d3021776e6833169c5e1.png", "sri":"sha256-+VfVTyNvfByi7t0zq33BBb1mLO5iuIBvsso4Ye2ysXw="}, "/assets/admin/ui-bg_glass_65_ffffff_1x400.png":{"target":"/assets/admin/ui-bg_glass_65_ffffff_1x400-b624f702075cd719a38f428e143025ea.png", "sri":"sha256-JFJf1ebnJmze7uXUNtoLcgn+dTh+bJxVxXVLIuQDGrU="}, "/assets/admin/ui-bg_gloss-wave_35_f6a828_500x100.png":{"target":"/assets/admin/ui-bg_gloss-wave_35_f6a828_500x100-23932de7c235b03187b8a5de3d024490.png", "sri":"sha256-Av4RPyWlnMm1at+9VqlutTAvTKW7637sDlQamg2ozNA="}, "/assets/admin/ui-bg_highlight-soft_100_eeeeee_1x100.png":{"target":"/assets/admin/ui-bg_highlight-soft_100_eeeeee_1x100-0b708185ce8927f18c7b3b82a7e7c247.png", "sri":"sha256-80rH2tcJybpprH1zkHIN1U/aVhUcZOc9mv9OEYavhRA="}, "/assets/admin/ui-bg_highlight-soft_75_ffe45c_1x100.png":{"target":"/assets/admin/ui-bg_highlight-soft_75_ffe45c_1x100-53acc69aaee6e2cb73b2021317e24af3.png", "sri":"sha256-JRY+0684rYlKOQKRQmXYJ5YiPhHS3kNPZmUVa3xi3hg="}, "/assets/admin/ui-icons_222222_256x240.png":{"target":"/assets/admin/ui-icons_222222_256x240-4c27b34156b7a3776f31cc456ca01c10.png", "sri":"sha256-/htyYBLdV3XU9kp9QnMKIQ8pBX6OgU8zkE05EsTZq9s="}, "/assets/admin/ui-icons_228ef1_256x240.png":{"target":"/assets/admin/ui-icons_228ef1_256x240-2f257489d9600ddf56d195a179ebc9df.png", "sri":"sha256-Z8eq2yeIM45jXe3uMbiKqOqQjA2MKR20tWcBTVc6bC8="}, "/assets/admin/ui-icons_ef8c08_256x240.png":{"target":"/assets/admin/ui-icons_ef8c08_256x240-772a64d6df4b93d230e9b38e1ab7522a.png", "sri":"sha256-loEi+IH5oyL8PSSjAGPT549tcpGfRzIbFEaqPFhR8ck="}, "/assets/admin/ui-icons_ffd27a_256x240.png":{"target":"/assets/admin/ui-icons_ffd27a_256x240-6686d21b904e18b100210169bb1c04ad.png", "sri":"sha256-O+dEGxrQchuZxTdpQJQ4LEirQLT1OA5lBNYO9O8mJo8="}, "/assets/admin/ui-icons_ffffff_256x240.png":{"target":"/assets/admin/ui-icons_ffffff_256x240-a19fe1b2d726e920c12e43a65df9039a.png", "sri":"sha256-sVicDAn7JIIW+osNHhgSqa+bSRWOtS4G94BitP/MAds="}, "/assets/bootstrap-theme.css":{"target":"/assets/bootstrap-theme-0a435682ec3e7fe26afdf79c9558a6dd.css", "sri":"sha256-qr/tUcFr7gjSLM/fASBhq43GcBc+sYVghk9FthUFoCc="}, "/assets/bootstrap.css":{"target":"/assets/bootstrap-398412f60b889d9618cd624fdc913eb3.css", "sri":"sha256-B+Sl2vWro3ZdPh3ESlYJxTZoO4Kc26tSZzdjj/qNvzo="}, "/assets/bootstrap.js":{"target":"/assets/bootstrap-1772ab74c1f593547907dc74235f33da.js", "sri":"sha256-L4RhCgE0byFav8kWcYcSZ2SVGjqGx39F/0yripDIFg4="}}
```

Comment:
I regenerated this file to include the SRI attributes, but unlike the others, it resulted in the content being in a different order.